components/local-app/cmd/root.go

```diff
@@ -88,17 +90,33 @@ var rootCmd = &cobra.Command{
 			return err
 		}
 		cmd.SetContext(config.ToContext(context.Background(), cfg))
+
+		telemetryEnabled := !telemetry.DoNotTrack()
+		telemetryEnabled = telemetryEnabled && cfg.Telemetry.Enabled
+		// For now we only enable telemetry on gitpod.io
```

Comment:
❤️ 

@filiptronicek we should be careful it then enabling for Dedicated to report to the proper host